src/domains/receipts-common/03_storage.tf

```diff
@@ -84,6 +84,8 @@ module "receipts_datastore_fn_sa" {
   blob_delete_retention_days = var.receipts_datastore_fn_sa_delete_retention_days
   tags                       = var.tags
 
+  blob_last_access_time_enabled = true
```

Comment:
to try it , we need to change this
`"git::https://github.com/pagopa/terraform-azurerm-v3.git//storage_account?ref=v6.7.0"`
in
`"git::https://github.com/pagopa/terraform-azurerm-v3.git//storage_account?ref=PRDP-54-feat-introduce_last_access_time_in_storage"`

Reply:
Done